tarantool_test.go

```diff
@@ -790,6 +790,27 @@ func TestNetDialer_BadUser(t *testing.T) {
 	}
 }
 
+// NetDialer does not work with PapSha256Auth, no matter the Tarantool version
+// and edition.
+func TestNetDialer_PapSha256Auth(t *testing.T) {
+	authDialer := AuthDialer{
+		Dialer:   dialer,
+		Username: "test",
+		Password: "test",
+		Auth:     PapSha256Auth,
+	}
+	ctx, cancel := test_helpers.GetConnectContext()
+	defer cancel()
+	conn, err := authDialer.Dial(ctx, DialOpts{})
+	if conn != nil {
+		conn.Close()
+		t.Fatalf("Connection created successfully")
+	}
+
+	assert.NotNil(t, err)
+	assert.ErrorContains(t, err, "failed to authenticate")
```

Comment:
It should be enough.
```suggestion
	assert.ErrorContains(t, err, "failed to authenticate")
```

Reply:
Updated.